### ISSUE TITLE
Style -- update dropdown over-rides to match error colour

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/application-manager-web-app#813]
For example:

![Screen Shot 2021-06-24 at 12 57 34 PM](https://user-images.githubusercontent.com/5456533/123186238-d9b73e00-d4eb-11eb-875f-9e2bcb519b70.png)

**Comment by CarlosNZ** [comment]
Did you fix this? Cos the colours for Dropdown still look like the red one on your screenshot on #930.

![Screen Shot 2021-08-24 at 11 57 09 PM](https://user-images.githubusercontent.com/16461988/130612390-2347e067-d46d-4927-a123-ecf48bdd4df4.png)

**Comment by nmadruga** [comment]
Oh! I though the problem was about the error color. I'll add a PR to fix this now - and also the search with also is missing the error-handling. Trying to do too many things at the same time. 😝

**Comment by CarlosNZ** [comment]
The Dropdown should look consistent with the Text Input in the original screenshot (i.e. pink border, white inner, pointy error label)

![Screen Shot 2021-06-24 at 12 57 34 PM](https://user-images.githubusercontent.com/5456533/123186238-d9b73e00-d4eb-11eb-875f-9e2bcb519b70.png)

**Comment by nmadruga** [comment]
Is this better?
![Screen Shot 2021-08-25 at 6 03 41 PM](https://user-images.githubusercontent.com/16461988/130735111-e706f82f-8c33-4fe0-b41b-84cee33999e9.png)

Note: I still need to add some validation to the element - will do that while making other required changes to OrgLicence template today : )

**Comment by CarlosNZ** [comment]
Yeah looks good. How hard is it to get the placeholder text too? (Prob should just stay grey)

**Comment by nmadruga** [comment]
No hard - I'm making this change now.

I have also added the other fixes for validation to keep concise with also adding the fix to the Search plugin
